packages/pancake-uikit/src/theme/colors.ts

```diff
@@ -24,6 +24,7 @@ export const lightColors: Colors = {
   background: "#FAF9FA",
   backgroundDisabled: "#E9EAEB",
   backgroundAlt: "#FFFFFF",
+  backgroundAlt2: "rgba(255, 255, 255, 0.7)",
```

Comment:
If this is used as a one-off maybe leave it out of the colors object? If this is the same as `backgroundAlt` but with different opacity, you can extend the hash string to change the alpha - https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Reply:
Good point, I was wondering if the hex + opacity was well supported. I'll check this and switch if it is fine.

Reply:
Apparently, it's not supported in IE11: https://caniuse.com/css-rrggbbaa

Reply:
We use it in other places. Go ahead. IE11 is not, and should not, be supported.